web/src/main/java/com/epam/rft/atsy/web/controllers/LoginController.java

```diff
@@ -10,16 +15,29 @@
  */
 @Controller
 @RequestMapping("/login")
-public class LoginController {
-  private static final String VIEW_NAME = "login";
+public class LoginController implements ApplicationContextAware {
+  private static final String VIEW_NAME_LOGIN = "login";
+  private static final String VIEW_NAME_WELCOME = "secure/welcome";
+
+  @Autowired
+  AuthenticationService authenticationService;
```

Comment:
Please use `private` for dependencies. Spring and Mockito can inject dependencies into private fields using reflection.
Other code must not access the controller dependencies.